### ISSUE TITLE
Training issues

### ISSUE THREAD

**Issue by carolche** [HenriquesLab/ZeroCostDL4Mic#218]
Hi Everyone,

I'm experiencing the same problem as a previous user (mafsousa, March 2022). I tried training my own model on the Cellpose/Omnipose notebook as well as command line (from kevinjohncutler's github), but I've been getting errors.
I'm using the following:
- OS: Mac Monterey 12.6
- Browser: Chrome
- Cellpose notebook version 1.0.2, 2.0, 2.1
- with or without the most recent Omnipose
The shape of my images is (1024, 1024, 1). I created the masks using Fiji adaptive thresholding plugin and are saved as binary images with the same shape as the images.  
There was no problem with the paths and I was able to mount the images as they were shown correctly in step 3.1, and subsequent steps were also error-free.
However, when I start training the model, I encounter errors as follows:
<img width="1497" alt="Screen Shot 2022-10-27 at 10 55 47" src="https://user-images.githubusercontent.com/77370541/198173578-adae8f84-7a91-46e8-96d4-de5cd40700f2.png">
I also had the same error as mafsousa.
Any help would be greatly appreciated. Thanks.
Caroline

**Comment by guijacquemet** [comment]
Hi @carolche 

Many thanks for reaching out!
It looks like you may have an issue with your training dataset. You indicate that you saved your segmentation ground truth as binary images? Do you have only one structure to segment per image or multiple ?
If you have more than one structure (ie multiple cells,), Cellpose would require label images to train.

**Comment by carolche** [comment]
@guijacquemet Yes, you are entirely right! I forgot to label the images! But alas, I still get an error.......
<img width="1274" alt="Screen Shot 2022-11-01 at 15 02 18" src="https://user-images.githubusercontent.com/77370541/199169099-fe54224f-cffb-4605-adcd-be8d12ba0bb8.png">
Thanks for your help.

**Comment by guijacquemet** [comment]
Hi @carolche

The error message shows you have less than 5 labels per image in your training dataset. Is this the case?

Maybe you could share your dataset?

Cheers

Guillaume

**Comment by carolche** [comment]
Hello @guijacquemet 

so sorry for the late reply! I really appreciate your help.
I was able to figure what the problem was. I wasn't labeling the masks appropriately. I tried a few different things and what worked best was to use ImageJ, adaptive threshold to make a binary mask, then threshold again, followed by counting and creating an ROI map. and it's working so far.....
Thank you for all your help! Keep up the good work!

Caroline

**Comment by Burt3** [comment]
Hi @carolche,

Would you please be able to provide an explanation to how you solved this issue (FileNotFoundError), as I am experiencing the exact same issue! I am currently producing a PNG mask image using the Cellpose GUI to complement my source images. 
Thanks:)
